backend/src/config/theme.rs

```diff
@@ -12,9 +12,20 @@ pub(crate) struct ThemeConfig {
     pub(crate) header_height: u32,
 
     /// Logo used in the top left corner of the page. Using SVG logos is recommended.
-    /// See the documentation on theming/logos for more info!
-    #[config(nested)]
-    pub(crate) logo: LogoConfig,
+    /// You can configure specific logos for small and large screens, dark and light mode,
+    /// and any number of languages. Example:
+    ///
+    /// ```
+    /// logos = [
+    ///     { path = "logo-large.svg", resolution = [425, 182] },
+    ///     { path = "logo-large-en.svg", lang = "en", resolution = [425, 182] },
+    ///     { path = "logo-large-dark.svg", mode = "dark", resolution = [425, 182] },
+    ///     { path = "logo-small.svg", size = "narrow", resolution = [212, 182] },
+    /// ]
+    /// ```
+    ///
+    /// See the documentation on theming/logos for more info and additional examples!
+    pub(crate) logos: Vec<LogoDef>,
```

Comment:
This example would fail with the `Duplicate logo definition` error below, wouldn't it? Ah no it wouldn't because of `*` mhhh...

So I would make this example config fail. I mentioned that when I described this "model" to you. In your example, it's unclear which logo should be used for the small dark version: the third or fourth entry in this array? This ambiguity I would like to solve by disallowing these configs. Judging from the filenames, the first three logos in your example are large ones, so I would require the `size = "wide"` key.

To be precise: for each case (combination of `(size, mode, lang)` there should be exactly one array entry that applies. Your check for "duplicate logo definition" is incorrect right now as `"*"` is different from `"narrow"`, so it doesn't detect that the 3rd and 4th line conflict. And as far as I see, there is also currently no check to make sure that for each, case there is at least one entry.

I would use confique `validate` feature, i.e. `#[config(validate = check_logos)]` and add `fn check_logos(logos: &Vec<LogoDef>) -> Result<(), String>`. 

If you want, I can write this function, it sounds kinda fun to me. But of course you can do it as well, just wanted to offer :P